cogs/sherpmail.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from discord.ext import commands
+from discord import app_commands
+
+# I couldn't work out replying directly to the initial messages so this is not the best
+# Nonetheless it is nearly 6am and I felt the need to complete this
+# I hope it works out and nobody abuses it lol good luck
+
+class sherpMailbox(commands.Cog):
```

Comment:
```suggestion
class SherpMailbox(commands.Cog):
```
(style)

Reply:
Good call! Applied to the incoming commit.